lib/whitehall_importer/edition_history.rb

```diff
@@ -6,14 +6,18 @@ def initialize(revision_history)
       @revision_history = revision_history
     end
 
-    def state_event(state)
+    def state_event!(state)
       event = revision_history.select { |h| h["state"] == state }.last
 
       raise AbortImportError, "Edition is missing a #{state} event" unless event
 
       event
     end
```

Comment:
I'd suggest adjusting this to be:

```
state_event(state) || raise AbortImportError, "Edition is missing a #{state} event"
```

so we don't need to duplicate the logic.

It's probably worth doing the same bang equivalents on the create_event and next_event methods to for consistency.